tests/console/glibc_i686.pm

```diff
@@ -1,30 +1,28 @@
 # SUSE's openQA tests
 #
 # Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2016 SUSE LLC
+# Copyright © 2012-2017 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Rework the tests layout.
+# Maintainer: Alberto Planas <aplanas@suse.com>
 
 use base "consoletest";
 use testapi;
+use utils 'zypper_call';
 use strict;
 
 # this part contains the steps to run this test
 sub run() {
     select_console 'root-console';
 
-    assert_script_run("zypper -n in -C libc.so.6", 500);
-
-    # select user console for our needles to match
-    select_console 'user-console';
-    script_run("/lib/libc.so.*", 0);
-    assert_screen 'test-glibc_i686-1';
+    zypper_call 'in -C libc.so.6';
+    script_run("/lib/libc.so.* | tee /dev/$serialdev", 0);
+    wait_serial(['GNU C Library (GNU libc) stable release', 'Configured for i686-suse-linux.']) || die '/lib/libc.so.* did not match';
```

Comment:
This is not the best way, but I was not successful to create regexp for this two lines and didn;t want to spend more time on it maybe somebody else can help